README.md

```diff
@@ -102,7 +102,7 @@ You can run this documentation offline by using [Docsify](https://docsify.js.org
 
 A PDF of all of the lessons can be found [here](https://microsoft.github.io/Web-Dev-For-Beginners/pdf/readme.pdf)
 
-## Other Curricula
+## Other Curricula/Curriculum
```

Comment:
We've used "Curricula" here as it is the plural form of "Curriculum", since it is referring to a list of many, so no need to include this updated text.

Reply:
okay thanks for clearing my misunderstanding